### ISSUE TITLE
Affective incontinence

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7417]
**Preferred term label:**
Affective incontinence

**Synonyms**

**Definition (free text, please give PubMed ID)**
-- define whether this is specific enough to be an HPO term

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID:27423618   OMIM: 221820 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
The original article that uses the term 'affective incontinence' is refering to pseudobulbar affect -- Emotional lability (pseudobulbar affect) is the abnormal expressions of emotion sometimes described in patients with pseudobulbar palsy, such as pathological laughter and emotional outburst, which is characterized by involuntary, uncontrollable laughter, or crying with paradoxical poverty of facial expression.
https://www.ncbi.nlm.nih.gov/books/NBK553160/

Proposed parent -[Abnormal emotion/affect behavior HP:0100851 ]( https://hpo.jax.org/app/browse/term/HP:0100851)

**Comment by pnrobinson** [comment]
@bcoleman96  -- does this merit its own term?

**Comment by bcoleman96** [comment]
I don't think so. Would be a good synonym though.

**Comment by pnrobinson** [comment]
I think this is a synonym of this existing term
[Pseudobulbar behavioral symptoms HP:0002193](https://hpo.jax.org/app/browse/term/HP:0002193)
It definitely deserves its own term -- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3849173/
I will revise and add synonyms

**Comment by pnrobinson** [comment]
done
